### ISSUE TITLE
List of repos / organizations does not refresh regularly

### ISSUE THREAD

**Issue by warthurton** [github/VisualStudio#6]
If you fork a report on github after you've already logged into the VS Plugin, when you try to clone the new repo it doesn't show up in the list.  Doing some additional testing (deleting a repo, removing an organization) show that those aren't update either.

I believe the list of repos/organizations should refresh each time you open the clone repo window.

I would dig into the code, but the Community addition does not allow you to work on VS extensions.

**Comment by shana** [comment]
@warthurton Thanks for the report! The list of repos gets loaded from cache and periodically refreshed, but the refresh rate might be a bit on the long side, which ends up looking like it never refreshes. I'll take a look.

VS community edition is what I use to develop, so you can definitely use it to build extensions :smile: With 2015, there are no more restrictions in what you can do in VS, it is the equivalent of Pro in previous VS versions.

**Comment by warthurton** [comment]
Thanks for the info on VS Community.  I just assumed and you know what happens then...

**Comment by shana** [comment]
Another user report about this. I'm bumping the priority on it so it's next on the list of things to fix.

**Comment by AndrewSav** [comment]
If it helps at all that "another user reported", please have yet another report - mine =). Fixing efforts are much appreciated!

**Comment by shana** [comment]
Working on it!

**Comment by tannert44** [comment]
Is there a fix for this yet?

**Comment by shana** [comment]
@tannert44 Not yet, but working on it!

**Comment by lendrick** [comment]
Is there a work-around for this?  If not, how long do I need to wait for the cache to refresh?

**Comment by shana** [comment]
@lendrick I've ran more tests and looks like the cache refresh is really broken for repositories :cry: 
I'm bumping the priority on this one, but in the meantime, you can kill the cache by shutting down VS, going to `%APPDATA%\GitHubVisualStudio` and deleting the `*.cache.db` file corresponding to your server (for github.com, `api.github.com.cache.db`).

Make sure VS is not running when you do this, the extension might not like it if you yank the cache out from under it while it's running.

**Comment by jonburney** [comment]
+1 for this, I had the same issue today. I deleted the api.github.com.cache.db file and relaunched VS and the Github plugin was no-longer working or even visible at all. I'm using VS enterprise 2015. I restarted VS with the devenv.exe /log param and it worked fine after that

**Comment by shana** [comment]
@jonburney Could you send me the extension log file so I can see what happened? It's at `%LocalAppData%\GitHubVisualStudio\extension.log`. If you could send that to support@github.com with a link to this issue, that would be great.

**Comment by jonburney** [comment]
I've had a look and there's nothing useful in there. Just these two lines for today (with my username stripped).

```
2015-11-26 12:24:19.1012|INFO|thread: 1|RepositoryHost|Log in from cache for login '{USERNAME}' to host 'https://api.github.com/' SUCCEEDED
2015-11-26 12:31:38.0255|INFO|thread: 1|RepositoryHost|Log in from cache for login '{USERNAME}' to host 'https://api.github.com/' SUCCEEDED
```

I'll try to reproduce again and get more detailed log data for you

**Comment by antoops** [comment]
Hi Shana, is this bug fixed or planned to fix in future version? 
I suggest, if the repos are not refreshing loading time, at least giving a refresh button will give a manual  sync with github. Now I am going ahead with logout-login

**Comment by shana** [comment]
A fix for this issue will be included in release 1.0.16.1, which will be out before Monday. 

The vsix package is already available at https://github.com/github/VisualStudio/releases/tag/v1.0.16.1. If you could install it and test and let me know how it goes, that would be great!
